substrate/zombienet/0003-block-building-warp-sync/test-block-building-warp-sync.zndsl

```diff
@@ -13,18 +13,20 @@ charlie: reports peers count is at least 3 within 60 seconds
 dave: reports peers count is at least 3 within 60 seconds
 
 
-# db snapshot has 12133 blocks
-dave: reports block height is at least 1 within 60 seconds
-dave: reports block height is at least 12132 within 60 seconds
-dave: reports block height is at least 12133 within 60 seconds
+# db snapshot has {{DB_BLOCK_HEIGHT}} blocks
+alice: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
+bob: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
+charlie: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
 
-alice: reports block height is at least 12133 within 60 seconds
-bob: reports block height is at least 12133 within 60 seconds
-charlie: reports block height is at least 12133 within 60 seconds
+dave: reports block height is at least 1 within 60 seconds
+dave: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
```

Comment:
we may also want to check other validators are not stuck:
```suggestion
alice: reports block height is greater than {{DB_BLOCK_HEIGHT}} within 60 seconds
bob: reports block height is greater than {{DB_BLOCK_HEIGHT}} within 60 seconds
charlie: reports block height is greater than {{DB_BLOCK_HEIGHT}} within 60 seconds
```

Reply:
Done